compose/config/config_schema_v2.0.json

```diff
@@ -248,7 +248,8 @@
           },
           "additionalProperties": false
         },
-        "internal": {"type": "boolean"}
+        "internal": {"type": "boolean"},
+        "labels": {"$ref": "#/definitions/list_or_dict"}
```

Comment:
Now that config_schema_v2.1.json has been merged, we can move this property there.